contracts/cw1-subkeys/src/contract.rs

```diff
@@ -29,23 +27,27 @@ use crate::msg::{
 };
 use crate::state::{Allowance, Permissions, ALLOWANCES, PERMISSIONS};
 
+#[cfg(feature="boot")]
+use boot_contract_derive::boot_contract;
+
 // version info for migration info
 const CONTRACT_NAME: &str = "crates.io:cw1-subkeys";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
-#[cfg_attr(not(feature = "library"), entry_point)]
+#[cfg_attr(feature="boot", boot_contract)]
 pub fn instantiate(
-    mut deps: DepsMut,
+    deps: DepsMut,
     env: Env,
     info: MessageInfo,
     msg: InstantiateMsg,
 ) -> StdResult<Response> {
+    let mut deps = deps;
```

Comment:
We'll need to document this limitation somewhere (create an issue for it)

Reply:
We can discuss that in : https://github.com/AbstractSDK/BOOT/issues/94